src/nvim/mbyte.c

```diff
@@ -792,7 +792,7 @@ int utfc_ptr2char(const char_u *p, int *pcc)
 int utfc_ptr2char_len(const char_u *p, int *pcc, int maxlen)
 {
 #define IS_COMPOSING(s1, s2, s3) \
-  (i == 0 ? UTF_COMPOSINGLIKE((s1), (s2)) : utf_iscomposing((s3)))
+  (i == 0 ? utf_composinglike((s1), (s2)) : utf_iscomposing((s3)))
```

Comment:
```suggestion
  (i == 0 ? utf_composinglike(s1, s2) : utf_iscomposing(s3))
```
It's not a macro anymore, so it's not necessary

Reply:
it was needed because `IS_COMPOSING` is a macro. but that one is literally used in one place, so I will inline it.